tests/harness.ps1

```
@@ -469,6 +469,52 @@ $cpDecline      = $AllText -match [regex]::Escape('post-execution checkpoint (%H
+Set-Content -LiteralPath $pctFixture -Value $pctScript -Encoding Ascii
+$pctOutput = ''
+$pctExit = 1
+try {
+  $pctOutput = (& cmd.exe /d /v:off /c $pctFixture | Out-String).Trim()
+  $pctExit = $LASTEXITCODE
+} catch {
+  $pctOutput = "EXCEPTION: $($_.Exception.Message)"
+}
+Remove-Item -LiteralPath $pctFixture -Force -ErrorAction SilentlyContinue
+$hasPctSanitizer = ($pctExit -eq 0) -and ($pctOutput -eq 'prefix_SECRET_suffix')
+Write-Result 'batch.dll_bundle.pct_sanitizer' 'CLAUDE.md Item 24: the %VAR:%%=_% doubled-percent idiom used by the DLL-bundle loops _SAFE display sanitization genuinely strips a literal percent sign under a real cmd.exe (live-executed fixture, not static pattern matching -- Blinter E021 flags this construct but its own analyzer does not model the doubled-%% substitution escape)' $hasPctSanitizer @{ output=$pctOutput; exitCode=$pctExit }
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Write-Result' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)